listenbrainz/db/dump_manager.py

```diff
@@ -182,9 +200,57 @@ def create_incremental(location, threads, dump_id):
 
         # Write the DUMP_ID file so that the FTP sync scripts can be more robust
         with open(os.path.join(dump_path, "DUMP_ID.txt"), "w") as f:
-            f.write("%s %s incremental\n" % (end_time.strftime('%Y%m%d-%H%M%S'), dump_id))
+            f.write("%s %s incremental\n" %
+                    (end_time.strftime('%Y%m%d-%H%M%S'), dump_id))
+
+        current_app.logger.info(
+            'Dumps created and hashes written at %s' % dump_path)
+        sys.exit(0)
+
+
+@cli.command(name="create_feedback")
+@click.option('--location', '-l', default=os.path.join(os.getcwd(), 'listenbrainz-export'))
+@click.option('--threads', '-t', type=int, default=DUMP_DEFAULT_THREAD_COUNT)
+def create_feedback(location, threads):
+    """ Create a spark formatted dump of user/recommendation feedback data.
+
+        Args:
+            location (str): path to the directory where the dump should be made
+            threads (int): the number of threads to be used while compression
+    """
+    app = create_app()
+    with app.app_context():
+
+        end_time = datetime.now()
+        ts = end_time.strftime('%Y%m%d-%H%M%S')
+        dump_name = 'listenbrainz-feedback-{time}-full'.format(time=ts)
+        dump_path = os.path.join(location, dump_name)
+        create_path(dump_path)
+        db_dump.dump_feedback_for_spark(dump_path, end_time, threads)
+
+        try:
+            write_hashes(dump_path)
+        except IOError as e:
+            current_app.logger.error(
+                'Unable to create hash files! Error: %s', str(e), exc_info=True)
+            sys.exit(-1)
+
+        try:
+            if not sanity_check_dumps(dump_path, 3):
```

Comment:
it's difficult to understand what the value of the second argument is here, can we rename it to something like `check_expected_number_of_dumps` or make the expeted_count argument required, so that when reading this code we can understand its behavour?

Reply:
expected_count is required and documented. I don't think it needs a change.